src/renderer/components/HistoryProvider.tsx

```diff
@@ -122,6 +122,24 @@ export const HistoryProvider = ({ children }: React.PropsWithChildren<unknown>):
         [apply]
     );
 
+    useHotkeys(
+        'ctrl+z, cmd+z',
+        () => {
+            historyObj.history = historyObj.history.undo();
+            apply(historyObj.history.current);
+        },
+        [apply]
+    );
+
+    useHotkeys(
+        'ctrl+y, cmd+y',
```

Comment:
Please add 'ctrl+shift+z, cmd+shift+z' to this.

Reply:
oh yeah, duh